src/components/reusable/notification/notification.stories.js

```diff
@@ -12,12 +17,17 @@ const handleOverflowClick = (e) => {
   // overflow link click logic here to mark as unread
 };
 
+const onClose = (e) => {
+  action(e.type)(e);
+  // To close notification
+};
+
 export default {
   title: 'Components/Notification',
   component: 'kyn-notification',
   argTypes: {
     type: {
-      options: ['normal', 'clickable'],
+      options: ['normal', 'clickable', 'inline', 'toast'],
```

Comment:
Can't inline and toast also be clickable? What if we separate "clickable" from "type" and just infer it automatically if an `href` is provided.

Reply:
Nope inline and toast can't be clickable. Will cross check with Marco once.

Reply:
I mean here we already used `type="clickable"` inside notification panel
<img width="594" alt="Screenshot 2024-04-29 at 7 16 00 PM" src="https://github.com/kyndryl-design-system/shidoka-applications/assets/140634142/12d6c5f7-d31e-43b2-a389-e1f66767d126">

Reply:
Right, but it becomes irrelevant if you just check `if (this.href !== '')` or something instead